Detectors/FIT/FT0/simulation/src/Digits2Raw.cxx

```diff
@@ -0,0 +1,124 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "Headers/RAWDataHeader.h"
+#include "CommonDataFormat/InteractionRecord.h"
+#include "DataFormatsFT0/RawEventData.h"
+#include "FT0Simulation/Digits2Raw.h"
+#include <TStopwatch.h>
+#include <cassert>
+#include <fstream>
+#include <vector>
+#include <bitset>
+#include <iomanip>
+#include "TFile.h"
+#include "TTree.h"
+
+using namespace o2::ft0;
+
+ClassImp(Digits2Raw);
+//Digits2Raw(char * fileRaw, std::string fileDigitsName)
+void Digits2Raw::readDigits(const char* fileRaw, const char* fileDigitsName)
+{
+  std::cout << "**********Digits2Raw::convertDigits" << std::endl;
+  std::vector<o2::ft0::Topo> lut_data(NCHANNELS_PM * NPMs);
+  for (int link = 0; link < NPMs; ++link)
```

Comment:
Is this default initialization of the LUT? Can it go to a dedicated method of the class?